src/Microsoft.AspNet.Mvc.ModelBinding/Binders/CompositeModelBinder.cs

```diff
@@ -34,14 +33,17 @@ public CompositeModelBinder([NotNull] IEnumerable<IModelBinder> modelBinders)
 
         public virtual async Task<ModelBindingResult> BindModelAsync([NotNull] ModelBindingContext bindingContext)
         {
-            var newBindingContext = CreateNewBindingContext(bindingContext,
-                                                            bindingContext.ModelName);
-
+            var newBindingContext = CreateNewBindingContext(bindingContext, bindingContext.ModelName);
             var modelBindingResult = await TryBind(newBindingContext);
-            if (modelBindingResult == null && !string.IsNullOrEmpty(bindingContext.ModelName)
-                && bindingContext.FallbackToEmptyPrefix)
+
+            if (modelBindingResult == null &&
+                bindingContext.FallbackToEmptyPrefix &&
+                !string.IsNullOrEmpty(bindingContext.ModelName))
             {
-                // fallback to empty prefix?
+                // Remove any validation errors from first attempt; these are likely Required errors.
+                bindingContext.ModelState.Clear();
```

Comment:
right.  this shouldn't be needed anymore.  just a holdover from my previous fixes.